### DISCUSSION TITLE
Splitted zip file

### DISCUSSION THREAD

**Discussion by starkyuii** [anasty17/mirror-leech-telegram-bot#1020] [General]
How to unzip splitted zip file that is password protected 
Its has extension.zip.001 to .zip.002

**Comment by anasty17** [comment]
From zipped tg files ?

**Comment by starkyuii** [comment]
Sir let me explain
I had leeched 6gb password protected zip file.
its splited in 3parts of 2gb in zip format.
Now i want to extract the all the 3 zip files.I know the password but i don't know how to unzip all three .
Is it got currupted because of split?

**Reply by anasty17** [reply]
Not supported yet, but it's in todo list already

**Reply by starkyuii** [reply]
Ohk thanks sir
